docs/standard/frameworks.md

```diff
@@ -1,56 +1,144 @@
+| .NET MicroFramework | netmf | netmf |
+| Windows | win | win [win8, netcore45] |
+| | | win8 [netcore45, win] |
+| | | win81 [netcore451] |
+| | | win10 (not supported by Windows 10 platform) |
+| Silverlight | sl | sl4 |
+| | | sl5 |
+| Windows Phone | wp | wp [wp7] |
+| | | wp7 |
+| | | wp75 |
+| | | wp8 |
+| | | wp81 |
+| | | wpa81 |
+| Universal Windows Platform | uap | uap [uap10.0] |
+| | | uap10.0 |
+
+## Deprecated frameworks
+
+The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
+
+Deprecated framework | Replacement
+--- | ---
+aspnet50 | netcoreapp
+aspnetcore50 | 
+dnxcore50 | 
+dnx | 
+dnx45 | 
+dnx451 | 
+dnx452 | 
+dotnet | netstandard
+dotnet50 | 
+dotnet51 | 
+dotnet52 | 
+dotnet53 | 
+dotnet54 | 
+dotnet55 | 
+dotnet56 | 
+winrt | win
+
+## Precedence
+
+A number of frameworks are related to and compatible with one another but not necessarily equivalent:
+
+Framework | Can use
+--- | ---
+uap (Universal Windows Platform) | win81
+| | wpa81
+| | netcore50
+win (Windows Store) | winrt
+| | winrt45
+
+## .NET Standard
+
+The [.NET Standard](https://github.com/dotnet/standard) simplifies references between binary-compatible frameworks, allowing a single target framework to reference a combination of others. For background, see the [.NET Standard Library](library.md).
+
+The [NuGet Tools Get Nearest Framework Tool](https://aka.ms/s2m3th) simulates the NuGet logic used for the selection of one framework from many available framework assets in a package based on the project's framework. To use the tool, enter one project framework and one or more package frameworks. Click the **Submit** button. The tool indicates if the package frameworks you list are compatible with the project framework you provide.
+
+For NuGet 3.3 and earlier, use the `dotnet` series of monikers. For NuGet 3.4 and later, use the `netstandard` moniker syntax.
+
+## Portable Class Libraries
+
+For information on Portable Class Libraries, see the [Portable Class Libraries](https://docs.microsoft.com/nuget/schema/target-frameworks#portable-class-libraries) section of the *Target Framework* topic in the NuGet documentation.
+
+ Name | Description | .NET Standard
+ --- | --- | ---
+ monoandroid | Mono Support for Android OS | netstandard1.4
+ monotouch | Mono Support for iOS | netstandard1.4
+ monomac | Mono Support for OSX | netstandard1.4
+ xamarinios | Support for Xamarin for iOS | netstandard1.4
+ xamarinmac | Supports for Xamarin for Mac | netstandard1.4
+ xamarinpsthree | Support for Xamarin on Playstation 3 | netstandard1.4
+ xamarinpsfour | Support for Xamarin on Playstation 4 | netstandard1.4
+ xamarinpsvita | Support for Xamarin on PS Vita | netstandard1.4
+ xamarinwatchos | Xamarin for Watch OS | netstandard1.4
+ xamarintvos | Xamarin for TV OS | netstandard1.4
+ xamarinxboxthreesixty | Xamarin for XBox 360 | netstandard1.4
+ xamarinxboxone | Xamarin for XBox One | netstandard1.4
```

Comment:
What does this table do here? It doesn't seem to be related to the paragraph right before it. Shouldn't it have some sort of text introduction?